src/interpreter/InterpretNot.ts

```diff
@@ -17,15 +17,13 @@ export default function interpretNot(schema: Schema, model: CommonModel, interpr
   if (typeof schema.not === 'object') {
     const notSchema = schema.not;
     const newInterpreterOptions: InterpreterOptions = {
-      ...interpreterOptions, 
-      splitModels: false,
```

Comment:
I don't see this option in other "new" places. We will split models by default from now, is it a good?

Reply:
The reason why it is no longer located there is because we no longer try to keep a list of models that should be returned. The interpreter method ONLY tries to interpret the JSON Schema to a single CommonModel and that's it. 

The reason it was added was that the odd level `not` schema should at that time not be saved to a separate model. 

Does that make sense? 🤔

Reply:
Ok, so from this PR it will be split by default. :)

Reply:
It will yes, because at the time the post-interpreter is run, only the core CommonModel is present 🙂